README.md

```diff
@@ -24,6 +25,10 @@ In the case that a parameter is passed in multiple ways, priority is as follows:
 - Environment variable
 - System property
 - Properties file entry
+- SSM parameter
+
+Note:  The intention is to use SSM to securely store the OAuth client secret.  We recommend storing
+it as an SSM _secure string_.
```

Comment:
it might be difficult for someone to figure how to make this find a value from the SSM.  Maybe we can add some info on how to set that up?  I would image it would be something like this?

```
SYNAPSE_OAUTH_CLIENT_SECRET=/synapse-login-app/SynapseOauthClientSecret
```

That tells the login app to retrieve the secret stored in the AWS SSM key `/synapse-login-app/SynapseOauthClientSecret`

Reply:
To store a parameter the user would use the SSM [PutParameter API](https://docs.aws.amazon.com/systems-manager/latest/APIReference/API_PutParameter.html) with name= SYNAPSE_OAUTH_CLIENT_SECRET, value=<the secret>.  Would you like that in the ReadMe?

Reply:
good point, info on how to get parameter into the SSM.  I prefer command line way.  Here's a good readme that you might find inspiration from.. https://github.com/Sceptre/sceptre-ssm-resolver

Reply:
I would prefer not to duplicate AWS documentation (as in the sceptre doc') but rather to link to it.  That ensures that if their tools/APIs change that we are not publishing an out-of-date ReadMe.

Reply:
linking sounds fine to me.